web/src/stream_topic_history.ts

```diff
@@ -112,8 +112,8 @@ export class PerStreamHistory {
           topic would likely suffice, though we need to think about
           private stream corner cases).
         * pretty_name: The topic_name, with original case.
-        * historical: Whether the user actually received any messages in
-          the topic (has UserMessage rows) or is just viewing the stream.
+        * is_data_reliable: Whether the message_id field is absolutely
```

Comment:
I am not sure of the name here. Have just kept this for now and can update later.

Reply:
Yeah this seems fine; posted a comment about the name to use in other files.